daemon/algod/api/server/v2/utils.go

```diff
@@ -263,3 +265,17 @@ func convertToDeltas(txn node.TxnWithStatus) (*[]generated.AccountStateDelta, *g
 
 	return localStateDelta, stateDeltaToStateDelta(txn.ApplyData.EvalDelta.GlobalDelta)
 }
+
+// sanitizePrintableUTF8String checks to see if the entire string is a UTF8 printable string.
+// If this is the case, the string is returned as is. Otherwise, the empty string is returned.
+func sanitizePrintableUTF8String(in string) string {
```

Comment:
I feel like "sanitize" is the wrong connotation here, I'd expect that to return the input string but without a few bad parts. `printableOrEmpty()` ? `squashIfAnyNonPrintable()` ?

Reply:
Ideally I think this would return a bool and be called "isPrintableUTF8", but then you couldn't use it inline

Reply:
I like the `strOrNil` patterns we've used below as it avoids the testing patten.
The `printableUTF8OrEmpty` suggestion makes sense.
thanks!